RiotSwiftUI/Modules/Common/Avatar/Service/MatrixSDK/MXAvatarService.swift

```diff
@@ -19,12 +19,12 @@ import MatrixSDK
 import Combine
 import DesignKit
 
-enum AvatarServiceError: Error {
+enum MXAvatarServiceError: Error {
```

Comment:
Prefixes like `MX` are only needed for ObjC compatible libraries. Otherwise we can rely on module name for disambiguation. MatrixSDK.AvatarService.

Reply:
So we this is the Matrix implementation of the protcol within `Riot` target. Would you prefer `MatrixSDKAvatarService`? We also have a mock implementation `MockAvatarService`? and might have `KotlinAvatarService` / `RustAvatarService` in the future.

Reply:
For the default class we can use `AvatarService` for the moment like we do for other classes used in Swift in the Element app. When disambiguation will come we will think about having different targets if needed or different repos. `KotlinAvatarService` will be named `AvatarService.kt` and probably a bridge will be made to make it usable by `AvatarService.swift` so we should not make us some issues.